src/textures/bitmap.cpp

```diff
@@ -205,6 +230,53 @@ class BitmapTexture final : public Texture<Float, Spectrum> {
         m_mean = ScalarFloat(mean / pixel_count);
     }
 
+    /**
+     * Parse the channel string into a vector of channels and a new pixel format based on the new number of channels
```

Comment:
Should we move this to `libcore/bitmap.h` instead?

Reply:
I've added this as a pure helper function for parsing the channel string parameter in `src/textures/bitmap.cpp`. There is no declaration in the header file. I'm not sure if this should be a public reusable function in bitmap.h.